firmware/shared/controls/DI_driverTorqueRequest.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <cmath>
+#include <iostream>
+
+namespace ctrl {  
+
+// Templating means this function will work on any types that support addition.
+template <typename T, typename B>
+T DI_driverTorqueRequest(T READY_TO_DRIVE, T DI_Error, B AccelPedPos) {
+    T u1 = 0.0;

```

Comment:
since this variable is always 0, it is more clear to just return 0 and get rid of this value. This makes the code simpler to read since the reader doesn't need to mentally "remember" what the `u1` variable means.

This is an example of where we will take a C++ approach instead of using the exact simulink syntax.